app/emails/util.py

```diff
@@ -63,3 +64,8 @@ def send_or_handle_error(*args, **kwargs):
             }
         )
         abort(503, response='Failed to send email for event: {}'.format(error_desc))
+
+
+def escape_token_markdown(token):
```

Comment:
should we test this?

Reply:
Yes - I've included a test to ensure the same sequence we saw in the wild is being escaped properly.